### DISCUSSION TITLE
How to disable the Nvidia boot splash (and possibly replace it with a custom one) on Xavier NX

### DISCUSSION THREAD

**Discussion by brgl** [OE4T/meta-tegra#640] [Q&A] [answered]
How would one go about disabling the bootloader splash screen and maybe even replacing it with a custom image?

**Comment by lfdmn** [comment, marked as answer]
You need to make a new bmp.blob image. It is one of the many partitions written during the flash procedure.

There was a utility I used but cannot remember from the top of my head it's name. You'll find it in the NVidia forum.

**Reply by madisongh** [reply]
In recent L4T releases, at least, the tools are in `Linux_for_Tegra/tools/bmp-splash`.
